msteams-platform/concepts/capabilities-overview.md

```diff
@@ -21,12 +24,20 @@ Your Teams app have one or all of the following core capabilities:
 
 Your app can also take advantage of advanced capabilities, such as the [Microsoft Graph API for Teams](https://docs.microsoft.com/graph/teams-concept-overview).
 
-See the following illustration to get an idea which capabilities would provide the features you want in your app.
+The following illustration gives you an idea of which capabilities will provide the features you want in your app.
 
 :::image type="content" source="../assets/images/capabilities-overview.png" alt-text="Mind map illustrating what Teams app capabilities are.":::
 
-## Doing what's best for your users
+## Always consider your user
+
+As you familiarize yourself with Teams app development, you understand its core fundamentals. You understand that there is more than one way to build certain features. In such scenarios, consider how you can provide a more native experience to your user.
+For example, you can collect user input by with a form using an iframe. You can also do this using a task module, a dialog in Teams, which may provide a more native experience to your user.
```

Comment:
you can collect user input with a form. An additional by is in the sentence

Reply:
Also, this sentence did not make sense. form using an iframe - does this mean a tab? Because task modules can also be a form within an iframe. 

"For example, you can collect user input in a form built as a tab in the app. You can also do this using a task module without switching views and disrupting user's flow of work. It is important to choose extension points that provide least deviation from a user's regular flow of work".